pyatlan/model/assets/a_d_l_s_object.py

```diff
@@ -34,20 +34,45 @@
 class ADLSObject(ADLS):
     """Description"""
 
+    @overload
+    @classmethod
+    def creator(
+        cls,
+        *,
+        name: str,
+        adls_container_qualified_name: str,
+        adls_account_qualified_name: str,
+        connection_qualified_name: str,
+    ) -> ADLSObject: ...
+
+    @overload
+    @classmethod
+    def creator(
+        cls,
+        *,
+        name: str,
+        adls_container_qualified_name: str,
+    ) -> ADLSObject: ...
```

Comment:
I think the most common method should be first since that seems to be the order in the IDE